docker/hardhat.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:16.8.0-alpine
+
+WORKDIR /src
+
+COPY package.json /src/package.json
+COPY package-lock.json /src/package-lock.json
+
+COPY packages/hardhat/package.json /src/packages/hardhat/package.json
+COPY packages/hardhat/hardhat.config.js /src/packages/hardhat/hardhat.config.js
+
+RUN apk add --update python3 make g++ && rm -rf /var/cache/apk/*
```

Comment:
Tried out on my branch, and for the hardhat package this deps are not necessary:
```suggestion
```

Reply:
We have tried to remove this line, but something happened in the build step with Macbook M1. So, we decided to keep this line to have compatibility between M1 and non-M1 chips.